aconfigure.ac

```diff
@@ -155,10 +159,12 @@ dnl
 dnl Legacy macros
 dnl
 AC_SUBST(ac_target_arch)
+AC_SUBST(ac_stl_cpp_lib)
 case $target in
     *android*)
 	AC_DEFINE(PJ_ANDROID,1)
 	ac_target_arch=$TARGET_ABI
+	ac_stl_cpp_lib=$ANDROID_LIBS
```

Comment:
This may introduce problems as theoritically `ANDROID_LIBS` contains all the libs, instead of just the STL.
E.g: as in the makefile the libs will be copied to the app package, the APK size may get bloated due to those libs, while in this context it only needs the STL.